src/gui/editors/AutomationEditor.cpp

```diff
@@ -1055,58 +1058,55 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 
 	p.fillRect(0, TOP_MARGIN, VALUES_WIDTH, height() - TOP_MARGIN, m_scaleColor);
```

Comment:
I think this is the part that draw the gray areas, you might want to move the if statement above this